nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow_schema.json

```diff
@@ -5,195 +5,237 @@
+                },
+                "single_end": {
+                    "type": "boolean",
+                    "description": "Specifies that the input is single-end reads.",
+                    "fa_icon": "fas fa-align-center",
+                    "default": false,
+                    "help_text": "By default, the pipeline expects paired-end data. If you have single-end data, specify this parameter on the command line when you launch the pipeline. It is not possible to run a mixture of single-end and paired-end files in one run."
+                }
+            },
+            "required": [
+                "reads"
+            ],
+            "fa_icon": "fas fa-terminal"
         },
-        "tracedir": {
-            "type": "string",
-            "description": "Directory to keep pipeline Nextflow logs and reports",
-            "default": "./results/pipeline_info",
-            "fa_icon": "fas fa-cogs",
-            "hidden": true
+        "Reference genome options": {
+            "type": "object",
+            "properties": {
+                "genome": {
+                    "type": "string",
+                    "description": "Name of iGenomes reference.",
+                    "fa_icon": "fas fa-book",
+                    "help_text": "If using a reference genome configured in the pipeline using iGenomes, use this parameter to give the ID for the reference. This is then used to build the full paths for all required reference genome files e.g. `--genome GRCh38`."
+                },
+                "igenomes_base": {
+                    "type": "string",
+                    "description": "Directory / URL base for iGenomes references.",
+                    "default": "s3://ngi-igenomes/igenomes/",
+                    "fa_icon": "fas fa-cloud-download-alt",
+                    "hidden": true,
+                    "help_text": ""
+                },
+                "igenomes_ignore": {
+                    "type": "boolean",
+                    "description": "Do not load the iGenomes reference config.",
+                    "fa_icon": "fas fa-ban",
+                    "hidden": true,
+                    "default": false,
+                    "help_text": "Do not load `igenomes.config` when running the pipeline. You may choose this option if you observe clashes between custom parameters and those supplied in `igenomes.config`."
+                }
+            },
+            "fa_icon": "fas fa-dna"
         },
-        "igenomes_base": {
-            "type": "string",
-            "description": "Directory / URL base for iGenomes references",
-            "default": "s3://ngi-igenomes/igenomes/",
-            "fa_icon": "fas fa-cloud-download-alt",
-            "hidden": true
+        "Pipeline config options": {
```

Comment:
1-option groups are not really helpful I think. The interfaces are easier if these are just stand-alone properties.

Reply:
Like the `Reference genome options` this was in anticipation of more parameters being added specifically to this group:
https://github.com/drpatelh/nf-core-atacseq/blob/999c8b3925de895a9951422730d18d677a29e4ac/nextflow_schema.json#L337-L363

Dont mind moving it though because this may not always be the case. Which group do you reckon?

Reply:
Ok, yeah I see where you're coming from. Maybe we can group it with some of the other more random config parameters? I think of this one in the same group as "stuff that 99% of people are not going to care about", so could go alongside `monochrome_logs` etc perhaps?

Note that groups that have all-hidden parameters will themselves be hidden, so having 100% hidden groups is kind of nice.

Reply:
Moved https://github.com/nf-core/tools/pull/593/commits/3b320715c6e8df2ac819e3e84dfca594a39924d4